mongofiles/mongofiles.go

```diff
@@ -248,6 +261,22 @@ func (mf *MongoFiles) getTargetGFSFiles() ([]*gfsFile, error) {
 		if len(gridFiles) < len(mf.FileNameList) {
 			return nil, fmt.Errorf("requested files not found: %v", mf.FileNameList)
 		}
+	} else if mf.FileNameRegex != "" {
+		query := bson.M{
+			"filename": bson.M{
+				"$regex":   mf.FileNameRegex,
+				"$options": mf.StorageOptions.RegexOptions,
```

Comment:
What would the value be if user didn't provide options?

Reply:
It would be the empty string, which translates on the server side to standard PCRE.

Case in point, a command without a specified pattern would be equivalent to `/<pattern>/`, which is just a regex search for "<pattern>".

This is indicated by MQL reference spec that I linked above.

Reply:
If so, in the option definition, ```	RegexOptions string `long:"options" value-name:"<regex-options>" short:"o" description:"regex options used for get_regex"`
``` 
There should be a `default`empty value defined.

Reply:
Sounds good!